cookiesales.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+//Select the HTML element we will be modifing
+var displayResult = document.getElementById('tableResults');
+
+//Create our Object template
+function StoreLocations (storeName,hourlyMinCustomer,hourlyMaxCustomer,avegCookies) {
+  this.storeName = storeName;
+  this.hourlyMinCustomer = hourlyMinCustomer;
+  this.hourlyMaxCustomer = hourlyMaxCustomer;
+  this.avegCookies = avegCookies;
+  this.hourlyCookieSales = [];
+  this.storeHours = ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'];
```

Comment:
Since all stores have the same hours, consider moving storeHours to the global scope.  If stores will have differing hours, it is good to keep it on the object.